mloop/learners.py

```diff
@@ -1031,6 +1035,30 @@ def _rand2(self, index):
         '''
         r0, r1, r2, r3, r4 = self.random_index_sample(index, 5)
         return self.population[r0] + self.curr_scale * (self.population[r1] + self.population[r2] - self.population[r3] - self.population[r4])
+    
+    # custom evolutionary strategy
```

Comment:
We can delete this line since (assuming this PR is merged) these won't be custom any more!

```suggestion
```